src/routes/token/token.ts

```diff
@@ -14,20 +14,12 @@ export const tokenHandler: RequestHandler<
 > = async (req, res) => {
   const { refreshToken } = req.body;
 
-  // set expiresAt to now + 10 seconds.
-  // this means the refresh token is available for 10 more seconds to avoid race
-  // conditions with multiple request sent by the same client. Ex multiple tabs.
-  const expiresAt = new Date();
-  expiresAt.setSeconds(expiresAt.getSeconds() + 10);
-
-  // get user and set new expiresAt on the used refreshToken
   const refreshTokens = await gqlSdk
-    .getUsersByRefreshTokenAndUpdateRefreshTokenExpiresAt({
+    .getUsersByRefreshTokenOld({
       refreshToken,
-      expiresAt: expiresAt,
     })
     .then((gqlres) => {
-      return gqlres.updateAuthRefreshTokens?.returning;
+      return gqlres.authRefreshTokens;
     });
 
```

Comment:
I think we should also extend the expiration time of the refreshToken when its being used.

The default expiration time for a refresh token is 30 days. If the user keeps using the refresh token to get new access tokens the user will be signed out after 30 days because that's what the refresh token expires.

Ideally, the refresh token should reset its expiration time every time it's being used.

Does it make sense?

Reply:
Totally make sense. I will adjust to be refreshed when used to obtain new token.

I left the default expiration time to 30 days because I think:
1. 30 days expiration time by default is reasonable
2. Is configurable, so everybody can adjust it whatever they feel comfortable with (eg. 1 year)

Updating the Refresh token expiration is the key here, thanks!

Reply:
Addressed in https://github.com/nhost/hasura-auth/pull/156/commits/1c6fc32ff09af2991df067948e30878d5054e1fc

Reply:
Yea, the default expiration time should still be 30 days. Perfect.